pkg/diagnostics/http_monitoring.go

```diff
@@ -229,3 +231,54 @@ func (h *httpMetrics) FastHTTPMiddleware(next fasthttp.RequestHandler) fasthttp.
 		h.ServerRequestCompleted(ctx, method, path, status, respSize, elapsed)
 	}
 }
+
+func (h *httpMetrics) convertPathToMethodName(path string) string {
```

Comment:
We can make this helper method instead of receiver method as it is not using struct references. I am writing similar helper method too :-)

Reply:
my case will be little different, I need to extract the specific apipath..

Reply:
this is api path... so you can leverage it..